source/documentation/contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -0,0 +1,100 @@
+[[contributing-user-stories]]
+= Contribute user stories to oVirt documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Basic information about how to create user stories for oVirt GitHub repository
+
+toc::[]
+
+== Modularization backstory
```

Comment:
Suggestion: Either delete "backstory" or replace it with a phrase like "in oVirt and Red Hat Virtualization documentation".

Reply:
Rewrote it.